content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Multi-account and Multi-region Testing"
+weight: 8
+description: >
+  How to test multi-account and multi-region compatibility.
+---
+
+LocalStack supports multi accounts and regions compatibility. 
+This document describes how to test your changes against cross accounts and regions.
+
+We regularly run the circleci test jobs of the LocalStack community repository to check the compatibility of cross accounts against the changes. To achieve that, we have a [scheduled workflow](https://github.com/localstack/localstack/blob/master/.circleci/config.yml) on [LocalStack](https://github.com/localstack/localstack), which executes the tests with randomised account and region credentials every night at 1:00am UTC.
```

Comment:
I think this isn't really useful information here. Maybe we could move this down to the section on manually triggering the pipeline? Maybe rephrase the section below to "Test changes in CI with non-default credentials" (in contrast and right next to "Test changes locally with non-default credentials")?
Also this line contains multiple sentences (we try to have a single sentence each line, see best practices in the README).